### PR TITLE
Add example of using generate-hashes

### PR DESCRIPTION
**Pull request by pgrzesik**
Just a small documentation contrib:
- Added example of using generate-hashes flag

##### Contributor checklist

- [ ] Provided the tests for the changes NOT NEEDED
- [ ] Added the changes to CHANGELOG.md NOT NEEDED
- [ ] Requested (or received) a review from another contributor

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,13 +35,14 @@ $ pip-compile
 # This file is autogenerated by pip-compile
 # To update, run:
 #
-#    pip-compile --output-file requirements.txt setup.py
+#    pip-compile --output-file requirements.txt requirements.in
```
Comment (vphilippon) [inline README.md]: This was an example to use `pip-compile` straight with a setup.py file, without a `requirements.in`, so the example should be kept as before.

**Comment by vphilippon** [comment]
Hi @pgrzesik, sorry for the delay, this one fell into a crack on my side.
Rebasing on master should fix the build.

**Comment by pgrzesik** [comment]
Hey @vphilippon, sorry about the change of setup.py to requirements, I totally messed that up. Should be fixed now.

Regards,
Piotr

**Comment by vphilippon** [comment]
Thanks @pgrzesik !

**Pull request merged**